init-tools.cmd

```diff
@@ -46,15 +46,17 @@ if exist "%DotNetBuildToolsDir%" (
 echo Running %0 > "%INIT_TOOLS_LOG%"
 
 set /p DOTNET_VERSION=< "%~dp0DotnetCLIVersion.txt"
-if exist "%DOTNET_CMD%" goto :afterdotnetrestore
 
 :Arg_Loop
 if [%1] == [] goto :ArchSet
-if /i [%1] == [x86]         ( set ARCH=x86&&goto ArchSet)
+if /i [%1] == [x86]         ( set ARCH=x86)
+if /i [%1] == [-Debug-Intrinsics] ( set DOTNET_VERSION=3.0.100-alpha1-009622)
```

Comment:
We should probably have a better way of maintaining/tracking this version than having it embedded in a `.cmd` file. Maybe another file like `DotnetCLIVersion.txt`?